src/layman/map/util.py

```diff
@@ -278,6 +292,15 @@ def abort_map_chain(workspace, mapname):
     celery_util.abort_publication_chain(workspace, MAP_TYPE, mapname)
 
 
+def abort_map_chain_by_uuid(uuid):
+    info = layman_util.get_publication_info_by_uuid(uuid, context={'keys': ['_workspace', 'name']})
+    if not info:
```

Comment:
It is expected to have `info` and it is very strange to not have one, so remove this check.